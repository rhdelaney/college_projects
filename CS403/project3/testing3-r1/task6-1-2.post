(println "trying (pfs 2 3 7 23)")
(stream-display 17 (apply pfs '(2 3 7 23)))
(println)
