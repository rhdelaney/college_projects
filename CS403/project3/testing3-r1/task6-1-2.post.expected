trying (pfs 2 3 7 23)
(2 3 4 6 7 8 9 12 14 16 18 21 23 24 27 28 32 ...)
